### ISSUE TITLE
TabView - tab headers do not change foreground color within changing theme mode

### ISSUE THREAD

**Issue by nick4real** [enisn/UraniumUI#660]
![image](https://github.com/enisn/UraniumUI/assets/67540024/e5c83a1c-5238-45d2-a8c2-91118bee0f74)
![image](https://github.com/enisn/UraniumUI/assets/67540024/7bbf581f-e03e-4bcb-a7d1-00913377b2ea)
![image](https://github.com/enisn/UraniumUI/assets/67540024/ff4aa26e-e393-423f-9320-38aaca3951ab)
Headers becomes visible again after they been selected and deselected once like on last screenshot with tab one.
Is there any way to manually reload a foreground colors? Thanks

**Comment by nick4real** [comment]
As I can see, not the foreground color only, but header generally.
![image](https://github.com/enisn/UraniumUI/assets/67540024/edcc7f54-2a52-4c9b-99a5-5154bdceedc0)
![image](https://github.com/enisn/UraniumUI/assets/67540024/4abcf2d5-119a-4d20-9e10-784d6757e070)

**Comment by enisn** [comment]
Yeah, previously I was using AppThemeColor but after migrating .NET 8, MAUI started throwing exception when I try to use app theme binding:

https://github.com/enisn/UraniumUI/blob/eda026f667f5e297d0231f2ebc4f9b17c589434a/src/UraniumUI.Material/Controls/TabView.cs#L59-L61

I couldn't find a good way to make it without compromising performance so far

**Comment by enisn** [comment]
I'll try to find a workaround for runtime theme changing
